app/viewmodels/journeys/taskList/NationalInsuranceContributionsViewModel.scala

```diff
@@ -26,22 +28,27 @@ import viewmodels.journeys.{SummaryListCYA, getJourneyStatus}
 
 object NationalInsuranceContributionsViewModel {
 
-  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages): SummaryList = {
+  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus], taxYear: TaxYear)(implicit messages: Messages): SummaryList = {
 
-    val nicRow = buildRow(NationalInsuranceContributions, nationalInsuranceStatuses, dependentJourneyIsFinishedForClickableLink = false)
+    val isAdjustmentsAnswered = JourneyStatus.getJourneyStatus(Adjustments, nationalInsuranceStatuses).isCompleted
```

Comment:
Adjustments isn't a part of nationalInsuranceStatuses. 
There will only ever be one status for national insurance, so we can change this in the tasklist model in the FE and BE from `nationalInsuranceStatuses: List[JourneyNameAndStatus]` to just `nationalInsuranceStatus: JourneyNameAndStatus`.

We'll then need to pass in the completedTrades into this method from the TaskListController to check if Adjustments is completed for each business

Reply:
Done :)